src/replace-libraries-build-builder.ts

```diff
@@ -0,0 +1,34 @@
+import {versions} from './angular-versions';
+import {AngularJson} from './types/angular-json';
+
+export const pre10_1LibraryBuilder = '@angular-devkit/build-ng-packagr:build';
+export const post10_1LibraryBuilder =
+  '@angular-devkit/build-angular:ng-packagr';
+
+export function replaceLibrariesBuildBuilder(
+  angularVersion: string,
+  angularJson: AngularJson
+): AngularJson {
+  const modifiedAngularJson = {...angularJson};
+  for (const key in angularJson.projects) {
+    if (Object.prototype.hasOwnProperty.call(angularJson.projects, key)) {
```

Comment:
suggestion: You could replace this logic with a for-of statement but I would prefer using `Object.entries` with `Array#filter` to filter the library projects with a build target.